css-syntax/styles.css

```diff
@@ -0,0 +1,33 @@
+body {
+  font-family: sans-serif;
+  color: #090302;
+}
+
+h1 {
+  text-align: center;
+}
+
+article {
+  padding-left: 16px;
+  padding-right: 16px;
+  border: 1px solid;
+  border-radius: 6px;
+  margin-bottom: 16px;
+}
+
+.title {
+  font-family: monospace;
+  font-size: 24px;
+}
+
+#html {
+  color: red;
+}
+
+#css {
+  color: green;
+}
+
+#js {
+  color: blue;
+}
```

Comment:
Oh good, the first CSS exercise was especially designed to teach students bad habits.

It's fine for the exercise, since those were the instructions, but in real-world usage it's almost never a good idea to use ids as a CSS selector instead of classes.